tests/test_pcp.py

```diff
@@ -0,0 +1,37 @@
+"""This module contains the tests for the pcp container
+"""
+import pytest
+import time
+from pytest_container.runtime import LOCALHOST
+
+from bci_tester.data import PCP_CONTAINER
+from bci_tester.runtime_choice import DOCKER_SELECTED
+
+
+@pytest.mark.parametrize("container", [PCP_CONTAINER], indirect=True)
+@pytest.mark.skipif(DOCKER_SELECTED, reason="only podman is supported")
+def test_systemd_present(container):
+    """Check that the pcp daemons are running.
+
+    """
+
+    # pcp needs a little time to initialize
+    time.sleep(5)
```

Comment:
Beware, this will break on slow hosts. Have you considered using a HealthCheck instead? pytest_container supports waiting until the container is healthy.

Reply:
nope, this was just a quick hack to make it pass. I'll look into the HealthCheck - thanks

Reply:
Do you mean `healthcheck_timeout_ms`? That waits until the container is started?  But here we need to wait until pcp services are running (not initializing). Also with docker it seems you can give your container image a healthcheck command, but with podman you can only to provide it on run.